File: april-30-days/day01-05-best-time-to-buy-and-sell-stock-ii.go
package main

func maxProfit(prices []int) int {
	total := 0
	maxIndex := len(prices) - 1
	for i := 0; i < maxIndex; i++ {
		if prices[i+1] > prices[i] {
			total += prices[i+1] - prices[i]
		}
	}
	return total
}

File: 2019/april-30-days/day01-05-best-time-to-buy-and-sell-stock-ii.js
var maxProfit = function(prices) {
    let total = 0
    let maxIndex = prices.length - 1
    for(let i = 0; i < maxIndex; i++) {
      if(prices[i+1] > prices[i]){
        total += prices[i+1] - prices[i]
      }
    }
    return total;
}
